mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -4,12 +4,20 @@ CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 	FIELD field_28646 shot Z
 	FIELD field_33399 owner Lnet/minecraft/class_1297;
 	METHOD method_24919 setProperties (Lnet/minecraft/class_1297;FFFFF)V
-		ARG 1 user
+		COMMENT Sets velocity and rotation.
+		ARG 1 shooter
+			COMMENT the entity who shot this projectile; used to add the shooter's velocity
+			COMMENT to this projectile
 		ARG 2 pitch
+			COMMENT the pitch
 		ARG 3 yaw
+			COMMENT the yaw
 		ARG 4 roll
-		ARG 5 modifierZ
-		ARG 6 modifierXYZ
+			COMMENT the roll
+		ARG 5 speed
+			COMMENT the length of the velocity
+		ARG 6 divergence
+			COMMENT the multiplier for the random vector added to the direction
```

Comment:
Maybe mention this is like fuziness, where player usages have 1 and other mobs/tools have higher values like 12 for snow golem, 8 for witch, 6 for dispensers, and difficulty-adjusted value for skeleton shooting, etc.

Reply:
I'm happy to rename it to `fuzziness` if it's clearer. Well, we got three candidates: `divergence`, `variation` and `fuzziness`.